addons/sms/static/src/js/fields_phone_widget.js

```diff
@@ -8,11 +8,35 @@ var session = require('web.session');
 var _t = core._t;
 
 /**
- * Override of FieldPhone to use add a button calling SMS composer if option activated
+ * Override of FieldPhone to use add a button calling SMS composer if option activated (default)
  */
 
 var Phone = basic_fields.FieldPhone;
 Phone.include({
+    /**
+     * By default, enable_sms is activated
+     *
+     * @override
+     */
+    init() {
+        this._super.apply(this, arguments);
+        this.enableSms = this.attrs.options.enable_sms === undefined ? true : this.attrs.options.enable_sms;
+        // reinject in nodeOptions (and thus in this.attrs) to signal the property
+        this.attrs.options.enable_sms = this.enableSms;
```

Comment:
Otherwise studio will be confused and won't take the option as absent as trueish.
https://github.com/odoo/enterprise/blob/13.0/web_studio/static/src/xml/view_editor_sidebar.xml#L485